prow/jobs/console/react-components/react-components.yaml

```diff
@@ -0,0 +1,28 @@
+presubmits:
+  kyma-project/console:
+    - name: console-react-components
+      run_if_changed: "^components/react/"
+      branches:
+        - prow-react-components
+      skip_report: true
+      decorate: true
+      path_alias: github.com/akucharska/console
+      extra_refs:
+        - org: kyma-project
+          repo: test-infra
+          base_ref: master
+          path_alias: github.com/kyma-project/test-infra
+      labels:
+        preset-dind-enabled: "true"
+        preset-sa-gcr-push: "true"
+        preset-docker-push-repository: "true"
+        preset-build-pr: "true"
```

Comment:
That job only needs `preset-build-pr: "true"`, others are not needed as that job doesn't build Docker image